src/typed-method-types/apps.ts

```diff
@@ -187,8 +187,18 @@ export type DatastoreQueryArgs<
      * @description The name of the datastore
      */
     datastore: Schema["name"];
+    /**
+     * @description A query filter expression
+     * @see {@link https://api.slack.com/automation/datastores-retrieve#filter-expressions}.
+     */
     expression?: string;
+    /**
+     * @description A map of attributes referenced in expression
```

Comment:
```suggestion
     * @description A map of attributes referenced in `expression`
```

(most IDEs will render markdown-like formatting)